tool/src/org/antlr/v4/tool/ErrorType.java

```diff
@@ -522,15 +522,15 @@
 	 */
 	USE_OF_BAD_WORD(134, "symbol <arg> conflicts with generated code in target language or runtime", ErrorSeverity.ERROR),
 	/**
-	 * Compiler Error 134.
+	 * Compiler Error 183.
```

Comment:
💭 Wasn't this part of another pull request?

Reply:
Yes, it is.